Source/JavaScriptCore/runtime/IntlLocale.cpp

```diff
@@ -226,6 +226,28 @@ void IntlLocale::initializeLocale(JSGlobalObject* globalObject, JSValue tagValue
     initializeLocale(globalObject, tag, optionsValue);
 }
 
+// https://tc39.es/proposal-intl-locale-info/#sec-weekday-to-string
+static String weekDayToString(const String fw)
```

Comment:
- This should be `weekday`, not `weekDay`.
- The param type should be either `const String&` or `StringView`; the latter is preferred, so let's use it.